doc/lua_api.txt

```diff
@@ -856,6 +857,14 @@ A positional sound will only be heard by players that are within
 * e.g. `{name = "default_place_node", gain = 1.0, pitch = 1.0}`
 
 
+Special sound files
+-------------------
+
+These sound files are played back by the engine if provided.
+
+ * `main_menu`: Looped sound in the main menu (gain = 1.0)
+ * `player_damage`: The own player takes damage (gain = 0.5)
+ * `player_falling_damage`: The own player takes damage by falling (gain = 0.5)
```

Comment:
`The own player takes damage` doesn't make sense in English. Maybe also add something like `Played when ...`?

Reply:
Played when the own player takes damage.
?

Reply:
That's fine for me.

Reply:
'the local player' is good for me.